css-view-transitions-1/Overview.bs

```diff
@@ -1364,6 +1376,10 @@ urlPrefix: https://wicg.github.io/navigation-api/; type: interface;
 			1. Set |capture|'s [=captured element/old mix-blend-mode=] to the [=computed value=] of 'mix-blend-mode' on |element|.
 
 			1. Set |namedElements|[|transitionName|] to |capture|.
```

Comment:
We probably need to redefine `transitionName` in this loop. Can simply be the computed value for `view-transition-name` on element.

Reply:
Done.